var/spack/repos/builtin/packages/scantailor/package.py

```diff
@@ -19,10 +19,10 @@ class Scantailor(CMakePackage):
 
     version(
         "0.2.7",
-        sha256="3e27647621d43638888a268902f8fa098b06a70a5da5d0623b1c11220a367910",
+        sha256="0262079c15a0068412edfb7d6c2ed03159dc71eada4e31e48ac500f924254eca",
     )
 
-    depends_on("qt@5:")
+    depends_on("qt@5.14.2")
```

Comment:
Is there a reason to use this very specific patch version?

Reply:
When using qt@5.15.2, the following error was output, so qt@5.14.2 is required to build this version.
     922  ./spack-stage/spack-stage-scantailor-0.2.7-hrfebeg7cnh3yb
             km65lw25o2elsppzmz/spack-src/filters/page_layout/ImageView.cpp:291
             :15: error: aggregate 'QPainterPath outer_outline' has incomplete
             type and cannot be defined
     923       QPainterPath outer_outline;
     924                    ^~~~~~~~~~~~~
     925  ./spack-stage/spack-stage-scantailor-0.2.7-hrfebeg7cnh3yb
             km65lw25o2elsppzmz/spack-src/filters/page_layout/ImageView.cpp:298
             :15: error: aggregate 'QPainterPath content_outline' has incomplet
             e type and cannot be defined
     926       QPainterPath content_outline;

Reply:
I think this may be overly specific. Also, on Github [I see the following](https://github.com/scantailor/scantailor/wiki/Building-from-Source-Code-on-Linux-and-Mac-OS-X) from `scantailor` repository:
```
Qt version 4.4.0 or higher (but Qt version 5.x won't work). If you have KDE 4.1 or later installed then you will have the runtime libraries but you may have to install the header files and utilities. These are in the package libqt4-dev on Ubuntu and Debian. Generally, developer packages have the -dev or -devel suffix.
```
but I am honestly not sure if the information there is outdated.

Reply:
When I built using qt version 4 series, the following error was output.
I thought I needed version 5 of qt.
     52    CMake Error at CMakeLists.txt:171 (FIND_PACKAGE):
     53      By not providing "FindQt5Core.cmake" in CMAKE_MODULE_PATH this project has
     54      asked CMake to find a package configuration file provided by "Qt5Core", but
     55      CMake did not find one.